boards/arm/segger_trb_stm32f407/segger_trb_stm32f407.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: segger_trb_stm32f407
+name: SEGGER-TRB-STM32F407
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 128
+flash: 1024
```

Comment:
```suggestion
ram: 192
flash: 512
```

Reply:
@martinjaeger side question, what is actually done the the `ram: 192` info? because the 192 includes the CCM ram and that is default not usable, so if a program/test needs 160k RAM and decides this board can run it because it has 192k things go wrong.

Reply:
Not sure what the info is used for, to be honest. The .yaml file in general is used for unit-testing. But the available memory during build is checked based on the information in the device tree.